coalib/results/Diff.py

```diff
@@ -381,13 +381,52 @@ def add_lines(self, line_nr_before, lines):
         self._changes[line_nr_before] = linediff
 
     def change_line(self, line_nr, original_line, replacement):
-        """
+        r"""
         Changes the given line with the given line number. The replacement will
         be there instead.
+
+        Given an empty diff object:
+
+        >>> diff = Diff(['Hey there! Gorgeous.\n',
+        ...              "It's nice that we're here.\n"])
+
+        We can change a line easily:
+
+        >>> diff.change_line(1,
+        ...                  'Hey there! Gorgeous.\n',
+        ...                  'Hey there! This is sad.\n')
+        >>> diff.modified
+        ['Hey there! This is sad.\n', "It's nice that we're here.\n"]
+
+        We can even merge changes within one line:
+
+        >>> diff.change_line(1,
+        ...                  'Hey there! Gorgeous.\n',
```

Comment:
actually *if* the lines have changed before and the new line is incompatible I could also pass: `Hey there! stupidshit.\n` and `Hello. :( stupidshit.\n` but I think if I do stupid shit before my change in the line the offset will be wrong.

I don't see this causing bugs, you're always supposed to give the original line. I guess we could fetch this from self.file automatically to be safe... wanna file an issue? That one should be totally easy